concept-anchor.md

```diff
@@ -0,0 +1,51 @@
+* Starting to explain InnerSource may be difficult unless one starts with a point decision makers are familiar with.
+* Agile/DevOps/Lean have already been heavily marketed and are commonly discussed terms in corporations, conferences, magazines etc.
+
+## Solutions
+### Agile
+There are several approaches that can be taken in the context of Agile depending on who's listening. The following main concepts are from [Wikipedia](https://en.wikipedia.org/wiki/Agile_software_development)
+* *Incremental and evolutionary processes* - Submitting pull requests for early feedback, or creating an issue for early stakeholder discussion aids this process, allowing ideas to be shaped and defined in the very early conceptual stages, thus being able to nourish good ideas and filter out unfeasible ideas before there are any significant costs associated with them.
+* *Efficient and face-to-face communication* - This part will be difficult as communications are not face to face. However they are efficient and tend to be very inclusive. It's possible to cite that this is how much of the software that runs the internet was successfully coordinated and built so there's some virtue to doing it the IS way, which is efficient but online communication. Another point of argument is that there's no better way to communicate your expectations than to actually implement it, making the process extremely effective
+* *Quality Focus* this is a major point for IS. Because of the expected submission by a variety of teams and individuals, extra care is taken to make sure that the submitted code is screened to be production quality. Comprehensive documentation for submitters, test coverage, CI/CD, static code analysis and other techniques are often use to screen code from strangers in the open source world, these techniques have great benefits for internal teams.
+
+##### [SCRUM](https://en.wikipedia.org/wiki/Scrum_%28software_development%29)
+Tshorten the example and make it more concrete for SCRUM, a popular agile methodology, one could cite the following scenario:
+Team A depends on Team B's code. Team A has a pressing priority for delivery, Team B has their own priorities, none of which is Team A, what to do? Using InnerSource, Team A can put the required code changes in to their own backlog and submit it to Team B, essentially removing block making things more "Agile".
+
+### DevOps
+There is no fixed definition of DevOps but much of it is focused around integrating Software Development and other dependencies such as Operations (auxiliary support such as servers, security etc.) and other business to reduce time required to deliver value. So where Agile focuses on the actual software development, DevOps is seen to incorporate all the other areas involving delivery.
+
+One major concept with DevOps is not to fear change, to be able to remove the risks so that experiments on the system can be comfortably performed.
```

Comment:
Suggestion: _not to fear change_ => _to embrace change_ 
It seems more direct.

Reply:
one can be apathetic to change, I want to keep "fear" somehow as it's a real and relatable emotion that holds things back.

Reply:
Fair enough.